lease/lessor.go

```diff
@@ -46,7 +46,8 @@ type DeleteableRange interface {
 // a lessor is the owner of leases. It can grant, revoke,
 // renew and modify leases for lessee.
 // TODO: use clockwork for testability.
-type lessor struct {
+// TODO: create an interface for Lessor?
+type Lessor struct {
```

Comment:
I am not sure how the interface should look like, so I simply create a struct for now. The complexity comes from the fact that only the leader should keep in charge of the keepAlive and expirations.